sites.yml

```
@@ -1538,6 +1538,15 @@ sites:
     maintainers:
       - "[Ivan G Torre](https://imagej.net/User:Ivangtorre)"
 
+  - name: "MultiStackReg"
+    id: "MultiStackReg"
+    url: "https://sites.imagej.net/MultiStackReg/"
+    description: >-
+      [MultiStackReg](https://sites.imagej.net/MultiStackReg/)
+      does registration of a stack based on the registration results of another stack. It uses Turboreg as the backend.  
+    maintainers:
+      - "[Kota Miura](https://imagej.net/User:Miura)"
```

URLs for user pages on the wiki changed with the switch to GitHub pages, so let's use the updated URL instead of relying on the redirect:

```suggestion
      - "[Kota Miura](https://imagej.net/people/cmci)"
```